Update compensation.md (#65)
dbt-labs/handbook: docs/compensation.md

**- How are international salary bands built differently from US bands?**

> While there are many nuances to global compensation, in an effort to maintain a scalable and competitive international compensation strategy, we now develop our international salary bands using a tier'd structure that anchors to US Salary Bands and apply a consistent premium or discount percentage for a given country based on the localized data sets. 

Reviewer: ```suggestion
While there are many nuances to global compensation, in an effort to maintain a scalable and competitive international compensation strategy, we develop our international salary bands using a tiered structure that anchors to US Salary Bands and apply a consistent premium or discount percentage for a given country based on the localized data sets. 
```

> **- How was each countries tier determined?**

Reviewer: ```suggestion
**- How was each country's tier determined?**
```

Country tiers and their associated percentages are developed use 3 data sets.
1. Radford Role Specific Data
2. Mercer - Comptryx Role Specific Data
3. Radford - Location Analytics Tool (Country Aggregate approach)

> Through these 3 data sets we were able to compile an average differential between the US and Country X. This analysis and approach allows us to better “fill in the gaps” where there is missing data and also allows us to create a consistent and scalable framework.

Reviewer: ```suggestion
Through these 3 data sets we were able to compile an average differential between the US and each country. This analysis and approach allows us to better “fill in the gaps” where there is missing data and also allows us to create a consistent and scalable framework.
```
